certificate-policy.md

```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 
-Before the CA authorizes a Delegated Third Party to perform a delegated function, the CA SHALL contractually require the Delegated Third Party to:
+While this policy promotes automation for the Certification Authorities, there still exists a need to allow a delegation of responsibility to U.S. Federal Government entities that form part of the U.S. Federal Government enterprise.  This policy allows for Registration Authorities to be used who may not be affiliated with the same organizational unit that is operating the Certificate Authority.  
 
-1. Meet the qualification requirements of Section 5.3.1, when applicable to the delegated function;
-2. Retain documentation in accordance with Section 5.5.2;
-3. Abide by the other provisions of these Requirements that are applicable to the delegated function; and
-4. Comply with (a) the CA's Certificate Policy/Certification Practice Statement or (b) the Delegated Third Party's practice statement that the CA has verified complies with these Requirements.
-5. Agree on which party is responsible for the annual audit of the delegated functions.
+A CA MAY designate an Enterprise Registration Authority (RA) to verify certificate requests from the Enterprise RA's affiliated U.S. Federal Government entity.  
 
-The CA MAY designate an Enterprise RA to verify certificate requests from the Enterprise RA's own organization.
 The CA SHALL NOT accept certificate requests authorized by an Enterprise RA unless the following requirements are satisfied:
 
-1. The CA SHALL confirm that the requested Fully-Qualified Domain Name(s) are within the Enterprise
-RA's verified Domain Namespace.
-2. If the certificate request includes a Subject name of a type other than a Fully-Qualified Domain Name, the CA SHALL confirm that the name is either that of the delegated enterprise, or an Affiliate of the delegated enterprise, or that the delegated enterprise is an agent of the named Subject. For example, the CA SHALL NOT issue a Certificate containing the Subject name of "Agency1" on the authority of "Agency2", unless the two agencies have a system sharing agreement.   This requirement applies regardless of whether the accompanying requested Subject FQDN falls within the Domain Namespace of "Agency2's" Registered Domain Name.
+1. The CA SHALL confirm that the requested Fully-Qualified Domain Name(s) are within the Enterprise RA's verified Domain Namespace(s) as registered in the .GOV (DotGov) and .MIL (DotMil) gTLDs Domain Name Registrars.  
+2. The CA SHALL confirm that the requested Fully-Qualified Domain Name(s) are not within any Domain Namespace(s) for any U.S. State, Local, Territorial, Native Sovereign Nations, or any other entities identified as a _Non-Federal Agency_ in the DotGov Domain Name Registrar per United States Code (U.S.C.) 41 CFR Part 102-173. 
```

Comment:
I again want to note that I see it as all-disadvantage for us to include the state/local/territorial/tribal limitation in the CP/CPS, even if we expect it as an operational limitation for the foreseeable future. 

It doesn't pose any risk to the web PKI for us to issue properly domain-validated certs to SLTT, and from the perspective of the community, they have to accept our root with the understanding that there's nothing technically constraining us from issuing for a SLTT .gov anyway.

Reply:
I'll update this part to match the modified info from other sections.  The constraint is on our boundaries and not overstepping those boundaries from a public law, funding, and perception aspect. 

The best middle ground is making it a should versus a shall...intent is established.

Reply:
updated in https://github.com/uspki/policies/pull/147/commits/ffa38b8bfd957371b7a172aadcdd1baee655ff38

along with other RA items